napalm/__init__.py

```diff
@@ -1,5 +1,6 @@
 import pkg_resources
 import sys
+from napalm.base import get_network_driver
```

Comment:
shouldn't we just move the code?

Reply:
It shouldn't matter...napalm.base.__init__ will be processed regardless (as get_network_driver will use code from napalm.base).